cluster/acsk.go

```diff
@@ -288,6 +288,7 @@ func (c *ACSKCluster) CreateCluster() error {
 	if !ok {
 		return errors.New("could not cast cluster create response")
 	}
+	c.modelCluster.ACSK.KubernetesVersion = castedValue.KubernetesVersion
```

Comment:
Past tense of cast is cast as well

Reply:
Going to add a PR about Alibaba Nodepools soon where I will address this, thanks:)